index.html

```diff
@@ -51,8 +51,127 @@ <h1 class="intro-heading">Glad to see you!</h1>
+                    <ul class="skills">
+                        <li>html</li>
+                        <li>Ruby on Rails</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <a class="see-project" href="#">See Project</a>
+            </div>  
+        </article>
+
+        <article class="project-articles">
+            <div class="textside">
+                <h2>Tonic</h2>
+                    <ul class="card-ul">
+                        <li>CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>Experiment content creation feature that allows users to add an existing story over the course of a day without spamming their friends</p>
+                    <ul class="skills">
+                        <li>html</li>
+                        <li>Ruby on</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <a class="see-project" href="#">See Project</a>
+ 
+            </div>
+            <div class="card">
+                <img src="/images/snapshot4.png">
+            </div>    
+        </article>
+
+    </section>
+
+    <section id="about">
+        <div>
+            <h2>About<br>Myself</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <div class="social-media">
+                <a href="#" class="social-para social-blue">LET'S CONNECT</a>                    
+                <ul>
+                    <li><a target="_blank" class="social-tag" href="https://www.github.com/joeveoper"><img src="/images/twitter.png"></a></li>
+                    <li><a target="_blank" class="social-tag" href="https://www.linkedin.com/in/yinka-olatunji-83aa36b5/"><img src="/images/linkedin.png"></a></li>
+                    <li><a target="_blank" class="social-tag" href="https://www.twitter.com/joeveoper"><img src="/images/medium.png"></a></li>
+                    <li><a target="_blank" class="social-tag" href="https://www.medium.com/"><img src="/images/github.png"></a></li>
+                    <li><a target="_blank" class="social-tag" href="https://www.angellist.com/"><img src="/images/angellist.png"></a></li>
+                </ul>
+                <a class="see-project" href="#">GET My Resume</a>
+            </div>
+            <h3>Languages</h3>
+            <ul class="about-skills" >
+                <li><img src="/images/js.png"> Javascript</li>
+                <li><img src="/images/html.png"> HTML</li>
+                <li><img src="/images/css.png"> CSS</li>
```

Comment:
- [ ] All your images are currently not displaying. remember to add a dot at the beginning of the images relative path URL

![image](https://user-images.githubusercontent.com/63473584/219901466-909418b4-f53a-42f8-a00d-6b8bac1531c3.png)

Reply:
dot(.) has been added as a prefix to all my image src urls